pkgs/applications/misc/alacritty/default.nix

```diff
@@ -119,6 +119,6 @@ in buildRustPackage rec {
     homepage = https://github.com/jwilm/alacritty;
     license = with licenses; [ asl20 ];
     maintainers = with maintainers; [ mic92 ];
-    platforms = [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" ];
+    platforms = [ "x86_64-linux" "i686-linux" "x86_64-darwin" "aarch64-linux" ];
```

Comment:
I think this could be generalized to 

```
platforms = lib.platforms.linux ++ lib.platforms.darwin;
```

Reply:
mmm... I don't understand. `lib.platforms` is different in release-19.03 and master. Why that?
release-19.03
```
    # back-compat aliases
    platforms = systems.forMeta;
```
master
```
    # back-compat aliases
    platforms = systems.doubles;
```

Reply:
alright, gotcha, there were some hacks removed in master

@Mic92 yup, that may work as well.